playbooks/support.md

```diff
@@ -168,14 +168,23 @@ with. People reporting problems are still learning the process too, so a respons
 This can be a lot more effective, and more in-line with Artsy's values, than "What you've reported isn't an
 incident, talk to your PM." Remember, we're all on the same team!
 
+### Incidents
+
+- Multiple simultaneous Intercom support requests from partners (payments not succeeding, login failures)
+- Auction artworks are not available before auction open
+- _Any_ automated alert that shows up in #incidents
+
 ### Non-incidents
 
 - Charge details for partners need updating
 - Single partner cannot access CMS
 - Artwork appearance on site ("How come it's not showing up in...")
 
-### Incidents
+### Urgent Non-incident
 
-- Multiple simultaneous Intercom support requests from partners (payments not succeeding, login failures)
-- Auction artworks are not available before auction open
-- _Any_ automated alert that shows up in #incidents
+- An auction bid which needs changing before an auction
+- [...]
+
+For issues like this it's best to contact the related PM, to ask for their advice. This is so that we can 
```

Comment:
It can be a little tricky to segment issues in this way, and I'd almost rather us delegate anything which might fall under "urgent non-incident" into treating it as an actual "incident" or having a product solution.

Things like auction bids needing changing and subscription issues are _known_ issues which require developer intervention. For these, we should be diligent about alerting the PM and delegating the fix/feature to a relevant product team (so, if it happens again it's the responsibility of the responsible team and not the on-call folks). This frees up those on-call to deal with `incidents`.